test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -2368,6 +2400,21 @@ private static class MyReferenceAssemblies
 
         internal static class NetFramework
         {
+            // .NET Framework 3.5 is the only version < 4.5.1 still in support by Microsoft. Support for .NET Framework 3.5
+            // ends on Jan 9, 2029. (https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework)
+
+            // Currently missing in https://github.com/dotnet/roslyn-sdk/blob/main/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs
+            // since https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies.net35 has been published.
+            internal static readonly ReferenceAssemblies Net35 = new ReferenceAssemblies(
+                    "net35",
+                    new PackageIdentity("Microsoft.NETFramework.ReferenceAssemblies.net35", "1.0.2"),
+                    Path.Combine("build", ".NETFramework", "v3.5"))
+                .WithAssemblyIdentityComparer(DesktopAssemblyIdentityComparer.Default)
+                .AddAssemblies(ImmutableArray.Create("mscorlib", "System", "System.Core", "System.Data", "System.Data.DataSetExtensions", "System.Xml", "System.Xml.Linq"))
+                .AddLanguageSpecificAssemblies(LanguageNames.CSharp, ImmutableArray.Create("Microsoft.CSharp"))
+                .AddLanguageSpecificAssemblies(LanguageNames.VisualBasic, ImmutableArray.Create("Microsoft.VisualBasic"))
```

Comment:
I wonder if https://github.com/dotnet/roslyn-sdk/tree/main/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing could define this in the future.

Reply:
It could. It was only omitted because net35 reference assemblies weren't available originally.

Reply:
https://github.com/dotnet/roslyn-sdk/pull/907